### PR TITLE
shutdown: add page

### PR DESCRIPTION
**Pull request by pxgamer**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/windows/shutdown.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+`shutdown /s /c "{{reason}}"`
+
+- Abort a shutdown sequence:
+
+`shutdown /a`
+
+- Shutdown a remote machine:
+
+`shutdown /m {{\\computer}}`
```
Comment (mfrw) [inline pages/windows/shutdown.md]: This seems unclear. Should we mention specify if its the computer name or hostname/IP?

Comment (pxgamer) [reply pages/windows/shutdown.md]: I guess I can add `using it's hostname or computer name:` or something along those lines.

Comment (sbrl) [reply pages/windows/shutdown.md]: Maybe we could update the example instead to `\\hostname`?

Comment (agnivade) [reply pages/windows/shutdown.md]: Yes, `\\hostname` is definitely clearer

File: pages/windows/shutdown.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+- Specify a timeout in seconds to wait before shutting down:
+
+`shutdown /s /t {{seconds}}`
+
+- Specify a comment for the shutdown reason:
+
+`shutdown /s /c "{{reason}}"`
+
+- Abort a shutdown sequence:
```
Comment (agnivade) [inline pages/windows/shutdown.md]: I am guessing this aborts an active shutdown when the timeout is yet to expire ? In that case, I think we can clarify it like "Abort a shutdown sequence whose timeout is yet to expire"

**Pull request merged**